tests/playwright/pages/editor-page.ts

```diff
@@ -40,7 +40,7 @@ export default class EditorPage extends BasePage {
 	 *
 	 * @return {Promise<void>}
 	 */
-	async gotoPostId( id: number|string = this.postId ) {
+	async gotoPostId( id: number|string = this.postId ): Promise<void> {
```

Comment:
You don't have to be explicit with the return value, it is inferred.

Reply:
Consistency